samples/face_reid/src/index_builder_client.py

```diff
@@ -0,0 +1,162 @@
+"""Index builder module."""
+import argparse
+import os
+import pathlib
+import shutil
+import time
+
+import cv2
+import hnswlib
+import numpy as np
+
+from samples.face_reid.utils import pack_person_id_img_n
+from savant.client import JpegSource, SinkBuilder, SourceBuilder
+from savant.utils.logging import get_logger, init_logging
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
```

Comment:
Do we really need all these arguments as script parameters? Why not set them as constants?